GundaGunde/GG00044.xml

```diff
@@ -0,0 +1,1356 @@
+                        <decoNote type="miniature" xml:id="d33">
+                                 
+                        <locus from="87v" to="88v"/>
+                        <desc><ref type="authFile" corresp="AT1036MercyCovenant"/></desc>
+                     </decoNote>
+                     </decoDesc>
+                     
+            
+         </physDesc>
+               </msPart>
+               <msPart xml:id="p2">
+                  <msIdentifier/>
+                  <msContents>
+                     <summary/>
+                     
+                     <msItem xml:id="p2_i1">
+                        <locus from="92ra" to="98vb"/>
+                        <title type="complete" ref="LIT5437MiracleCommandments"/>
+                        <textLang mainLang="gez"/>
+                     </msItem>
+                     </msContents>
+                     <physDesc>
+                        <objectDesc form="Codex">
+                           <supportDesc>
+                              <support>
+                                 <material key="parchment"/>
+                              </support>
+                              <extent>
+                                 <measure unit="leaf">6</measure>
+                                 <locus from="1ra" to="5ra"/>
+                                 <measure unit="leaf" type="blank">1</measure>
+                                 <locus target="#I"/>
+                              </extent>
+                           </supportDesc>
+                           <layoutDesc>
+                              <layout columns="2" writtenLines="22">
+                                 <ab type="pricking">Pricking and ruling are visible.</ab>
+                                 <ab type="pricking">Primary pricks and horizontal pricks are visible.</ab>
+                                 <ab type="ruling" subtype="pattern">1A-1A-1A1A-1A1A/0-0/0-0/C</ab>
+                                 <ab type="ruling">The upper line is written above the ruling,
+                                    the lower line is written above the ruling.</ab>
+                              </layout>
+                           </layoutDesc>
+                        </objectDesc>
+                        </physDesc>
+                  <msPart xml:id="p3">
+                     <msIdentifier/>
+                     <msContents>
+                        <summary/>
+                     
+                     <msItem xml:id="p2_i2">
+                        <locus from="1ra" to="5ra"/>
+                        <title type="complete" ref="LIT5051MenbabFelsata"/>
+                        <textLang mainLang="gez"/>
+                     </msItem>
+                     
+                     </msContents>
+               <physDesc>
+                  <objectDesc form="Codex">
+                     <supportDesc>
+                        <support>
+                           <material key="paper"/>
+                        </support>
```

Comment:
please add `<measure>` and `<locus>` to indicate which folia this codicological unit spans